tests/functional/cypress.js

```diff
@@ -0,0 +1,12 @@
+const cypress = require('cypress');
+
+cypress.run({
```

Comment:
The only other thing that I am concerned about is that fact that we should have a config for a dev env?.

So this `.run()` is used just to run the test but when developing we probably want `cypress.open({})` configured which would allow us actively and easily write the tests with visibility of changes. I am thinking `.open()` would have config such as not exiting when a test fails, auto picking up code changes and re running etc. Thoughts?

Reply:
Do our current selenium tests work like this?

The api for cypress.open differs from that of the run call - it doesn't have a headed option for instance. There's nothing stopping a developer running cypress directly if they want such flexibility. 

Personally, I find the way that open automatically re-runs tests after any change is kind of annoying, especially with something as long-running as the census.

Reply:
I think running the cypress UI seperately is fine.

Reply:
Our current selenium tests don't work like this..thats the point?. We said we are using cypress because it is easier to develop with but at the moment its just straight up replacing `chimp` as a runner?. Also if you did `cypress open` manually it wouldn't pick up the spec folder automatically etc.

With our current chimp set up, we can do things like `@watch` to re-run a test once you made changes.

I would suggest at least adding how to run cypress interactively `(cypress open)` to the README.

Reply:
The reason it has to be run in this way is its the only way to accommodate the headless behaviour desired. I can add a minimum cypress.json file to allow either open/run, but it'll mean some duplication between the two configs.